arista-dcs-7124S/config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Arista Switch",
+    "rules": [
+        {
+            "path": "version.Hardwaremodel",
+            "equals": "AristaDCS-7124S-F"
+        },
+        {
+            "path": "version.Internalbuildversion",
+            "equals": "4.13.7M-1877079.4137M.1"
```

Comment:
Was looking for unique identifiers, but you are right. Should be removed so we are not locked down to a specific build version.